dev/io.openliberty.session.6.1.internal/resources/io/openliberty/session/resources/SessionMessages.nlsprops

```diff
@@ -0,0 +1,31 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+#CMVCPATHNAME io.openliberty.session.6.1.internal/resources/io/openliberty/session/resources/SessionMessages.nlsprops
+#COMPONENTPREFIX SESN
+#COMPONENTNAMEFOR SESN Session
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+
+# Note:
+#Last used error meessage code was SESN8502I
+#
+#Use 8600 - 8699 for Servlet 6.0 and ABOVE 
+#
+#From Session 6.1 (EE11) and UP, add new message to this file
+#
+# -----------------------------------------------
+session.is.invalid=SESN8602E: The session is invalid.
+session.is.invalid.explanation=The current session has expired and is invalid.
+session.is.invalid.useraction=Try to login or obtain new session again.
+# -----------------------------------------------
+session.id.is.invalid=SESN8603E: The session Id is invalid.
+session.id.is.invalid.explanation=The session Id has changed since last accessed.
+session.id.is.invalid.useraction=The session Id has changed since last accessed. Try to login again.
```

Comment:
```suggestion
session.id.is.invalid.useraction=The session Id has changed since last accessed. Try to login again.
# -----------------------------------------------
# If update this file, synchronize the message ID with the Session 6.1 found here io.openliberty.session.6.0.internal/resources/io/openliberty/session60/internal/resources/SessionMessages.nlsprops
# New message ID should be in the Session 6.1 from now.
```

To be consistent with what you provided in the 6.0 resource file to point to where the messages are also updated.